packages/aws-cdk-lib/pipelines/README.md

```diff
@@ -976,6 +976,10 @@ Note that if you provide an existing pipeline, you cannot provide values for
 `pipelineName`, `crossAccountKeys`, `reuseCrossRegionSupportStacks`, or `role`
 because those values are passed in directly to the underlying `codepipeline.Pipeline`.
 
+### Use pipeline service role as default action role in pipeline
+
+If you don't want cdk helps you automatically create roles for each actions(like CodeBuild step ,etc.),you could set `usePipelineServiceRoleForActions` as true to use pipeline service role as default action role in Codepipeline if no action role provided.
```

Comment:
```suggestion
By default CDK automatically creates roles for each action (`CodeBuildStep`, etc).
If you prefer to use the pipeline service role as default instead, set the `usePipelineRoleForActions` property.
This will tell CDK to default to the pipeline service role in AWS CodePipeline if no action role is provided.
```